screens/cart.html

```diff
@@ -0,0 +1,213 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../style.css">
+    <link rel="stylesheet" href="../stylesheets/screens/productslist.css">
+    <title>Cart | ArtStation</title>
+</head>
+
+<body class="cart-body">
+    <div>
+        <nav class="nav-container d-flex">
+            <div class="nav-brand d-flex">
+                <button class="fa fa-bars btn btn-link nav-btn"></button>
+                <a href="./home.html">
+                    <h2 class="nav-brand-name">ArtStation</h2>
+                </a>
+            </div>
+
+            <div class="nav-brand d-flex">
+                <input type="text" placeholder="search for product">
+                <button class="btn btn-link nav-btn">Search</button>
+            </div>
+            <div class="nav-pills d-flex">
+                <ul class="list list-no-bullet children-center">
+                    <li class="list-item">
+                        <a href="../index.html" class="btn btn-link nav-btn ">LOGIN</a>
+                    </li>
+                    <li class="list-item">
+                        <a href="./home.html"><i class="fa fa-solid fa-home"></i></a>
+                    </li>
```

Comment:
Your logo already redirects to the home page. You don't need another button for doing the same thing.

Reply:
Yeah but I want to have both